app/views/devise/registrations/edit.html.erb

```diff
@@ -1,39 +1,41 @@
-<h2>Edit <%= resource_name.to_s.humanize %></h2>
-
-<%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f| %>
-  <%= devise_error_messages! %>
-
-  <div class="field">
-    <%= f.label :email %><br />
-    <%= f.email_field :email, autofocus: true %>
+<div class="form-container">
+<%= devise_error_messages! %>
+  <div class="panel panel-default">
+    <div class="panel-heading">
+      <h3>Edit <%= resource_name.to_s.humanize %></h3>
```

Comment:
no action necessary, just a comment: this is the type of situation where a decorator would be useful.